docker/api/swarm.py

```diff
@@ -407,11 +411,18 @@ def update_swarm(self, version, swarm_spec=None, rotate_worker_token=False,
             :py:class:`docker.errors.APIError`
                 If the server returns an error.
         """
+        if rotate_manager_unlock_key:
+            if utils.version_lt(self._version, '1.25'):
+                raise errors.InvalidVersion(
+                    'Rotate manager unlock key '
+                    'is only available for API version >= 1.25'
+                )
 
         url = self._url('/swarm/update')
         response = self._post_json(url, data=swarm_spec, params={
             'rotateWorkerToken': rotate_worker_token,
             'rotateManagerToken': rotate_manager_token,
+            'rotateManagerUnlockKey': rotate_manager_unlock_key,
```

Comment:
This will pass `rotateManagerUnlockkey=false` even to services with api < 1.25. Did you check that this is ok or do we need to completely exclude the key in that case?

Reply:
You're right. Totally missed that. Better to be safe and only include it when set to `True` and >= 1.25.